docs/src/xhtml/components/times/langs.xhtml

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Local Time</title>
+		<object data="tabs.xhtml"></object>
+		<style>
+			td {
+				padding: 2.25px 11px;
+			}
+		</style>
+	</head>
+	<body>
+		<header data-ts="TopBar"></header>
+		<main data-ts="Main">
+			<article>
+				<h1>Time</h1>
+				<section class="desc">
+					<p>TODO: This should go into a screenshot for the test suite!</p>
+				</section>
+				<section data-ts="Spirit">
+					<script type="text/edbml">
+						<table>
+							('da de-CH de en-GB en-US en es ' +
```

Comment:
cool thing could be to move the locale-detection code in a shared file between ui and docs, so this page could just use that during generation.

that way whenever we add new languages they will magically show up. I guess this all should belong on a separate page showing what languages we support for dates and such.

Reply:
Agreed, but let's call that a feature creep for now.